### ISSUE TITLE
Build error for GISS-GC with `dev/giss-gc-14` branch

### ISSUE THREAD

**Issue by jwallwork23** [fetch4/geos-chem#2]
On BlueHive, following the [instructions](https://github.com/fetch4/GISS-GC/blob/dev/giss-gc-14/doc/GISS-GC-DOC/giss-gc_tutorial.md), I get as far as the compile step, in which the GEOS-Chem CMake build is seen to be progressing nicely and it gets to 92% before crashing with
```console
Error copying Fortran module "mod/get_met_mod.mod".  Tried "mod/GET_MET_MOD.mod" and "mod/get_met_mod.mod".
```

**Comment by jwallwork23** [comment]
See also https://github.com/fetch4/GISS-GC/issues/2.

**Comment by joewallwork** [comment]
Closed by #3.
